assets/js/themes.js

```diff
@@ -46,11 +47,11 @@ function scheme_switch (e) {
     if (localStorage.getItem('dark_mode')) {
       return;
     }
-  } catch {}
+  } catch (exception) {}
```

Comment:
In this case `exception` used instead of just `e` because of `e` already used in outer scope. Anyway, this variable is never used (and this is a good reason to use it later in next PRs)